spec/factories.rb

```diff
@@ -281,12 +281,16 @@
   end
 
   factory :order_with_totals_and_distribution, parent: :order do
+    transient do
+      shipping_fee 3
+    end
+
     distributor { create(:distributor_enterprise) }
     order_cycle { create(:simple_order_cycle) }
 
-    after(:create) do |order|
+    after(:create) do |order, proxy|
       p = create(:simple_product, :distributors => [order.distributor])
-      FactoryBot.create(:line_item_with_shipment, :order => order, :product => p)
+      FactoryBot.create(:line_item_with_shipment, shipping_fee: proxy.shipping_fee, :order => order, :product => p)
```

Comment:
I would replace those hashrockets now that you are at it

Reply:
yep, done.